README.asc

```diff
@@ -16,7 +16,7 @@ There are two ways to generate e-book content from this source code.
 
 The easiest way is simply to let us do it. A robot is standing by to look for new work on the main branch and automatically build it for everyone.
 
-You can find the current builds on http://git-scm.com/book[] and more information about the builds available at https://progit.org[].
+You can find the current builds on http://git-scm.com/book[] in an online readable format. There is also a link to buy the book online in hardcopy (English only).
```

Comment:
I agree with @Cpt-xx here; it's good to remove the broken link, but we don't need to mention the hardcopies here. If a reader is looking for that, they'll find it with their own searching, or by following the link at the top right of every localized [book page](https://git-scm.com/book/nl/v2).

Regarding this readme, it might be worth looking into syncing with the [English edition's version](https://github.com/progit/progit2/blob/master/README.asc). It's been updated since this was forked with build instructions, removing this link, and other improvements.

Reply:
I will try and find some time to re-synch with the English edition. It has been a while since I've had a look.